index.Rmd

```diff
@@ -131,58 +124,57 @@ In its most basic use-case, `read_csv` expects that the data file:
 
 ```{r load_state_property_data, warning=FALSE, message=FALSE}
 library(tidyverse)
 read_csv("data/can_lang.csv")
 ```
-Above you can also see something neat that Jupyter does to help us understand our code: it colours text depending on its meaning in R. For example, 
-you'll note that functions get bold green text, while letters and words surrounded by quotations like filenames get blue text.
+
 
 > **In case you want to know more (optional):** 
-> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster and it creates a nicer variant of the base R data frame called a *tibble*. 
+> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster, and it creates a nicer variant of the base R data frame called a *tibble*. 
 > This has several benefits that we'll discuss in further detail later in the course.
 
-## Assigning value to a data frame
+## Giving a data frame a name
 
-When we loaded the language data collected in the 2016 Canadian census in R above using `read_csv`, we did not give this data frame a name, so it was 
-just printed to the screen and we cannot do anything else with it. That isn't very useful; what we would like to do is give a name to the data frame that `read_csv` outputs
-so that we can use it later for analysis and visualization. 
+When we loaded the language data collected in the 2016 Canadian census in R above using `read_csv`, we did not give this data frame a name. Therefore the data was just printed on the screen, and we cannot do anything else with it. That isn't very useful; we would like to give a name to the data frame that `read_csv` outputs to use later for analysis and visualization. 
 
-To assign name to something in R, there are two possible ways---using either the assignment symbol (`<-`) or the equals symbol (`=`). From a style perspective, 
-the assignment symbol is preferred and is what we will use in this course. When we name something in R using the assignment symbol, `<-`, we do not need to surround 
-it with quotes like the filename. This is because we are formally telling R about this word and giving it a value. Only characters and words that act as values need 
-to be surrounded by quotes.
+There are two possible ways to assign a name to something in R ---using either the assignment symbol (`<-`) or the equals symbol (`=`). From a style perspective, the assignment symbol is preferred and is what we will use in this course. When we name something in R utilizing the assignment symbol, `<-`, we do not need to surround it with quotes like the file name because we are formally telling R about this word and giving it a value. Only characters and words that act as values need to be surrounded by quotes.
```

Comment:
```suggestion
There are two possible ways to assign a name to something in R ---
using either the assignment symbol (`<-`) or the equals symbol (`=`). 
From a style perspective, the assignment symbol is preferred 
and is what we will use in this course. 
When we name something in R using the assignment symbol, `<-`, 
we do not need to surround the name we are giving (on `<-`'s left-hand side) 
with quotes like the file name 
because we are formally telling R about this new word and giving it a value. 
Only characters and words that act as values (on `<-`'s right-hand side) 
need to be surrounded by quotes.
```